packages/ui/v2/core/WizardForm.tsx

```diff
@@ -65,7 +67,7 @@ function WizardForm<T extends DefaultStep>(props: {
                   onClick={() => {
                     setStep(step + 1);
                   }}>
-                  {step < steps.length ? "Next" : "Finish"}
+                  {step < steps.length ? t("next_step_text") : t("finish")}
```

Comment:
Instead of introducing useLocale here. These strings should come directly form a prop.